src/services/app.ts

```diff
@@ -83,7 +86,12 @@ export async function makeEnvFile(options: InitOptions): Promise<string> {
 CLIENT_ID=${options.clientId}
 CLIENT_SECRET=${options.clientSecret}
 GWA_ENV=${options.env}
-API_VERSION=${options.apiVersion ?? '2'}
+API_VERSION=${options.apiVersion ?? '2'}${
```

Comment:
Just a readability recommendation, this would output a new line and be a bit cleaner.

```javascript
...
API_VERSION=${options.apiVersion ?? '2'}
${options.dataCenter ? `DATA_CENTER=${options.dataCenter}` : ''}
```